### ISSUE TITLE
Download Amphetamine without Apple's App store?

### ISSUE THREAD

**Issue by antdude** [x74353/Amphetamine-Enhancer#18]
Is it possible to download and install without Apple's App store like its Enhancer?

Thank you for reading and hopefully answering soon. :)

**Comment by sean-alex** [comment]
It seems the author answered the question in https://github.com/x74353/Amphetamine-Enhancer#readme in the "Why Not Un-Sandbox Amphetamine and Distribute it Outside the App Store?" section.

**Comment by raxod502** [comment]
I'm not sure I fully understand the reasoning in that section. The main point seems to be:

> As an independent developer, with little to no income from my apps, I don't have a desire or the financial resources to pay out-of-pocket for the bandwidth and web hosting fees that would be incurred by distributing Amphetamine outside of the App Store. I also don't have an advertising budget either.

But, posting releases on GitHub, which is a platform the developer is already using, would be free, since GitHub doesn't charge for bandwidth or hosting. GitLab and other alternative platforms are also generally free for this type of hosting. Having a second download location also wouldn't require any advertising.

I suppose the developer may simply not feel it is worth the time to publish releases in two places, which is reasonable. After all, there isn't any reason we should feel entitled to the developer's time.

But it certainly would make me very happy if there were a way to use the application without supporting Apple any more than necessary.

**Comment by alurm** [comment]
Yeah, it's a shame I can't download this app without an Apple account. I understand the developer though.

**Comment by EfeAtesh** [comment]
Ok, you don't need to download it from appstore.

 I found this alternative solution just paste this command to terminal:
`main@Mac ~ % sudo pmset -a disablesleep 1 `
Thus, it will disable the clamshell tracking and your Mac won't be in sleep mode. But keep in mind, paste this command when you want to reactivate the sleep clamshell tracking thing. It won't be reset unless you close it
`main@Mac ~ % sudo pmset -a disablesleep 0`

**Comment by owens-ben** [comment]
I completely understand and respect the developer's reasoning. However, I'm just adding my 👍 to this request. It would be amazing to have both options.
